### ISSUE TITLE
Cannot install CA certificate on PE 12 (Plus or non-plus)

### ISSUE THREAD

**Issue by silviuk** [PixelExperience/android-issues#2710]
## Build date

PixelExperience_Plus_dipper-12.1-20220521-1804-OFFICIAL

## Expected Behavior

I go to Settings -> Security -> Advanced settings -> Encryption and credentials -> Install a certificate -> CA certificate.
I accept "Your data won't be private" warning by clicking "Install anyway". I perform the requested biometric or PIN authorization.
I am allowed to select a certificate file from the system.
The certificate is installed and later trusted for the intended purposes (e.g. HTTPS traffic).

## Current Behavior

I go to Settings -> Security -> Advanced settings -> Encryption and credentials -> Install a certificate -> CA certificate.
I accept "Your data won't be private" warning by clicking "Install anyway". I perform the requested biometric or PIN authorization.
The process appears to crash (no error) and I'm taken back to the "Encryption and credentials" page.
If I try for a 2nd time the process, I now get an explicit error after the biometric or PIN authorization: "Certificate Installer keeps stopping (options: App info, Close app, Send feedback)"

Note: I have experienced this bug in all PE12 builds for dipper. It appears to have existed in PE 11 builds of dipper as well, as can be seen in https://github.com/PixelExperience/android-issues/issues/1188 and also PE 11 builds of other devices (Ginkgo) https://github.com/PixelExperience/android-issues/issues/1135, so it is likely a cross-device issue and not specific to dipper.

## Possible Solution

N/A.

## Steps to Reproduce

1. Go to Settings -> Security -> Advanced settings -> Encryption and credentials -> Install a certificate -> CA certificate.
2. Accept "Your data won't be private" warning by clicking "Install anyway". I perform the requested biometric or PIN authorization.
3. The process appears to crash (no error) and returns to the "Encryption and credentials" page.
4. Try for a 2nd time the steps 1-3. An explicit error is thrown after the biometric or PIN authorization: "Certificate Installer keeps stopping (options: App info, Close app, Send feedback)"

/device dipper
/version twelve_plus

**Comment by pixelexperiencebot** [comment]
Issue created! You can close at any time by commenting ```/close```

**Comment by jhenrique09** [comment]
Can you provide logs? Cannot confirm that issue

**Comment by silviuk** [comment]
Here are the logs and the zipped cert (cannot upload .crt to the issue):
[logcat.txt](https://github.com/PixelExperience/android-issues/files/8867317/logcat.txt)
[adguard_075502.zip](https://github.com/PixelExperience/android-issues/files/8867327/adguard_075502.zip)

Thank you!

**Comment by silviuk** [comment]
For the record: I don't even get to select which cert file to install. com.android.certinstaller crashes right after providing my pin/biometrics.

**Comment by silviuk** [comment]
Hi, I have not modded this rom at all and neither removed this package. As you can see, others reported this bug before, so I would appreciate if you can think of other reasons why DocumentsUI is/might appear as not available, thank you.

**Comment by silviuk** [comment]
Solved. 

Details:
First I tried to install an older DocumentsUI.apk, but it wouldn't work due to conflicting package names.
So then I looked for an app that can show all system apps. Running it, I found a DocumentUI.apk installed as system app*. After opening the app info page, I saw it was marked as disabled. Enabling it solved the problem, CA certificates can now be successfully installed.

*Google's own Files app wouldn't show it in its own system apps list, or at least I couldn't find it in the non-searchable list if shows.
